lib/tasks/install-blueprint.js

```diff
@@ -77,20 +77,26 @@ class InstallBlueprintTask extends Task {
   _handleLookupFailure(name, error) {
     logger.info(`Blueprint lookup for "${name}" failed`);
 
-    if (!validateNpmPackageName(name).validForNewPackages) {
+    // I wish I didn't have to do this https://github.com/npm/validate-npm-package-name/issues/22
```

Comment:
Hmm, is there a reason we don't use https://github.com/npm/npm-package-arg? It is _explicitly_ intending to be "parse anything you can pass to `npm install xxxx`", and internally uses `validate-npm-package-name`.

Reply:
fixed.